<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLink">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar link de youtube:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entryLink">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cargaLink">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar link</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonDescarga">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Descargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCargado">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Link cargado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
